rotkehlchen/accounting/structures/base.py

```diff
@@ -277,6 +279,17 @@ def get_identifier(self) -> str:
     def get_assets(self) -> list[Asset]:
         return [self.asset]
 
+    def has_details(self) -> bool:
+        return self.extra_data is not None and 'sub-swaps' in self.extra_data
+
+    def get_details(self) -> Optional[dict[str, Any]]:
+        details = {}
+
+        if self.extra_data is not None and 'sub-swaps' in self.extra_data:
+            details['sub-swaps'] = self.extra_data['sub-swaps']
+
+        return details if len(details) > 0 else None
```

Comment:
this still has the problem we said about the extra info in the liquity staking events right?